app/scripts/modules/netflix/pipeline/stage/chap/chapExecutionDetails.html

```diff
@@ -34,8 +34,11 @@
           <div>
             <a href="{{runUrl}}" target="_blank">Run Details</a>
           </div>
-          <div>
-            <a href="{{canaryReportUrl}}" target="_blank">Canary Report</a>
+          <div ng-repeat="canaryReport in canaryReports">
+            <a href="{{canaryReport.url}}" target="_blank">
+              Canary Report
+              <span ng-show="canaryReports.length > 1"> ({{ canaryReport.key }})</span>
```

Comment:
We typically go with `ng-if` to avoid rendering (`{{ canaryReport.key }}` gets rendered and watched here, for example), any use `ng-show` if we actually want the component to get initialized.